### PR TITLE
convert to v2

### PR DESCRIPTION
**Pull request by wojsmol**
fixes https://github.com/wp-cli/wp-super-cache-cli/issues/20
test results from my fork https://travis-ci.com/wojsmol/wp-super-cache-cli.

### CODE REVIEW

**Comment by wojsmol** [comment]
fixes https://github.com/wp-cli/wp-super-cache-cli/issues/20

**Comment by szepeviktor** [comment]
> fixes #20

It needs to be added in the top comment.

**Comment by wojsmol** [comment]
@szepeviktor Ives if github don't catch this then this will be reference for @schlessera :smile:

File: composer.json [hunk 1]
```diff
@@ -15,9 +15,33 @@
         "role": "Developer"
     }],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.4",
+        "wp-cli/extension-command": "2",
```
Comment (schlessera) [inline composer.json]: This is a testing requirement, it should be moved into `"require-dev"` section.

**Comment by schlessera** [comment]
Ah, clicked on the request review button too fast...

@wojsmol Please make the following additional changes:
* move `cli.php` into `src/WPSuperCache_Command.php`
* add PSR-4 autoloader that maps `src/` to an empty namespace
* add a `wp-cli.yml` file that requires the `wp-super-cache-cli.php` file

**Comment by wojsmol** [comment]
I'm temporally using in tests this link https://github.com/wojsmol/wp-super-cache-cli/archive/tov2.zip

**Pull request merged**